src/duplo_resource/jit.py

```diff
@@ -252,6 +253,10 @@ def update_aws_config(self,
     config = os.environ.get("AWS_CONFIG_FILE", f"{Path.home()}/.aws/config")
     cp = configparser.ConfigParser()
     cp.read(config)
+
+    # If name is not provided, use the global context variable or the subdomain from the --host flag
+    name = name or (self.duplo.context.get('name') if isinstance(self.duplo.context, dict) else None) or (urlparse(self.duplo.host or '').hostname or '').split('.')[0]
```

Comment:
Because of Duplo changing the portal URLs to all be custom under the users domain. We will need to drop the idea of using the subdomain as a name.